containers/ddev-gitpod-base/Dockerfile

```diff
@@ -36,6 +38,8 @@ RUN sudo apt-get clean && sudo rm -rf /var/lib/apt/lists/*
 # a gcc instance named gcc-5 is required for some vscode installations
 RUN sudo ln -sf $(which gcc) /usr/local/bin/gcc-5
 
+RUN mkdir -p ~/go/bin
+
 RUN for item in golang.org/x/tools/gopls@latest github.com/go-delve/delve/cmd/dlv@latest; do \
         go install $item; \
     done
```

Comment:
We should have `go clean -modcache` here in the same `RUN`, because `go install` downloads cache to `~/go/pkg`, and it's not removed.

Reply:
I wish we'd had this conversation earlier, I might have gotten out of the trouble I was in. I was really surprised about how this popped up when it didn't seem to have been there before. Perhaps upstream changed. Think it's sorted now.